 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 100-660-140
125.129.62.2 - - [11/Apr/2018 18:10:19] "GET / HTTP/1.1" 200 -
125.129.62.2 - - [11/Apr/2018 18:10:19] "GET /images/to.png HTTP/1.1" 404 -
223.33.164.33 - - [11/Apr/2018 18:22:57] "GET /detail/test130/45 HTTP/1.1" 200 -
66.102.6.176 - - [11/Apr/2018 20:18:40] "GET /favicon.ico HTTP/1.1" 404 -
66.102.6.174 - - [11/Apr/2018 20:18:40] "GET /favicon.ico HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:18:43] "GET / HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:18:43] "GET /css/main.css?q=1522674725 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:18:43] "GET /css/style.css?q=1522674725 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:18:43] "GET /images/to.png HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:18:44] "GET /favicon.ico HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:21:17] "GET /upload HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:21:17] "GET /bootstrap/3.0.0/css/bootstrap.min.css HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:22:20] "POST /uploaded HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:22:20] "GET /bootstrap/3.0.0/css/bootstrap.min.css HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:22:23] "GET /bootstrap/3.0.0/css/bootstrap.min.css HTTP/1.1" 404 -
Converting <Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.smi> OK!
Converting <Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.smi> OK!
122.47.22.201 - - [11/Apr/2018 20:26:08] "POST /uploaded HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:26:08] "GET /bootstrap/3.0.0/css/bootstrap.min.css HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:31:24] "GET / HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:31:24] "GET /images/to.png HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:31:24] "GET /images/myheroaca.jpg HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:31:38] "GET /detail/%EB%82%98%EC%9D%98%20%ED%9E%88%EC%96%B4%EB%A1%9C%20%EC%95%84%EC%B9%B4%EB%8D%B0%EB%AF%B8%EC%95%84%202%EA%B8%B0/66 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:31:38] "GET /images/to.png HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:31:39] "GET /detail/66/1?ani_path=%2Fuploads%2Fmy_hero_Academya2%2FOhys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:31:39] "GET /detail/66/jquery.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:31:39] "GET /detail/66/jquery.srt.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:31:39] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 20:31:39] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.vtt HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:31:39] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
122.47.22.201 - - [11/Apr/2018 20:31:39] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 20:31:41] "GET /detail/66/2?ani_path=%2Fuploads%2Fmy_hero_Academya2%2FOhys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:31:41] "GET /detail/66/jquery.srt.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:31:41] "GET /detail/66/jquery.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:31:41] "GET /detail/66/jquery.srt.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:31:41] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 20:31:41] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
122.47.22.201 - - [11/Apr/2018 20:31:41] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 20:31:44] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 20:31:46] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
66.102.6.23 - - [11/Apr/2018 20:44:38] "GET / HTTP/1.1" 200 -
66.102.6.23 - - [11/Apr/2018 20:44:38] "GET /favicon.ico HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:44:42] "GET / HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:44:42] "GET /images/to.png HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:44:44] "GET /detail/%EB%82%98%EC%9D%98%20%ED%9E%88%EC%96%B4%EB%A1%9C%20%EC%95%84%EC%B9%B4%EB%8D%B0%EB%AF%B8%EC%95%84%202%EA%B8%B0/66 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:44:44] "GET /images/to.png HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:46:36] "GET / HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:46:36] "GET /images/to.png HTTP/1.1" 404 -
121.129.145.110 - - [11/Apr/2018 20:47:03] "GET / HTTP/1.1" 200 -
121.129.145.110 - - [11/Apr/2018 20:47:03] "GET /css/main.css?q=1522674725 HTTP/1.1" 200 -
121.129.145.110 - - [11/Apr/2018 20:47:03] "GET /images/myheroaca.jpg HTTP/1.1" 200 -
121.129.145.110 - - [11/Apr/2018 20:47:03] "GET /css/style.css?q=1522674725 HTTP/1.1" 200 -
121.129.145.110 - - [11/Apr/2018 20:47:03] "GET /images/to.png HTTP/1.1" 404 -
121.129.145.110 - - [11/Apr/2018 20:47:04] "GET /images/to.png HTTP/1.1" 404 -
121.129.145.110 - - [11/Apr/2018 20:47:11] "GET /detail/%EB%82%98%EC%9D%98%20%ED%9E%88%EC%96%B4%EB%A1%9C%20%EC%95%84%EC%B9%B4%EB%8D%B0%EB%AF%B8%EC%95%84%202%EA%B8%B0/66 HTTP/1.1" 200 -
121.129.145.110 - - [11/Apr/2018 20:47:11] "GET /images/to.png HTTP/1.1" 404 -
121.129.145.110 - - [11/Apr/2018 20:47:15] "GET /detail/66/1?ani_path=%2Fuploads%2Fmy_hero_Academya2%2FOhys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
121.129.145.110 - - [11/Apr/2018 20:47:15] "GET /detail/66/jquery.srt.js HTTP/1.1" 404 -
121.129.145.110 - - [11/Apr/2018 20:47:15] "GET /detail/66/jquery.js HTTP/1.1" 404 -
121.129.145.110 - - [11/Apr/2018 20:47:16] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.vtt HTTP/1.1" 200 -
121.129.145.110 - - [11/Apr/2018 20:47:16] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
121.129.145.110 - - [11/Apr/2018 20:47:16] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
121.129.145.110 - - [11/Apr/2018 20:47:16] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
121.129.145.110 - - [11/Apr/2018 20:47:18] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
122.47.22.201 - - [11/Apr/2018 20:53:18] "GET / HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:53:18] "GET /detail/%EB%82%98%EC%9D%98%20%ED%9E%88%EC%96%B4%EB%A1%9C%20%EC%95%84%EC%B9%B4%EB%8D%B0%EB%AF%B8%EC%95%84%202%EA%B8%B0/66 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:53:18] "GET /images/to.png HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:53:20] "GET / HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:55:58] "GET / HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:58:29] "GET /detail/%EB%82%98%EC%9D%98%20%ED%9E%88%EC%96%B4%EB%A1%9C%20%EC%95%84%EC%B9%B4%EB%8D%B0%EB%AF%B8%EC%95%84%202%EA%B8%B0/1 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:58:29] "GET /images/to.png HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:58:30] "GET /detail/1/1?ani_path=%2Fuploads%2Fmy_hero_Academya2%2FOhys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 20:58:30] "GET /detail/1/jquery.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:58:30] "GET /detail/1/jquery.srt.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 20:58:30] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 20:58:30] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
122.47.22.201 - - [11/Apr/2018 20:58:30] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 21:00:24] "GET /detail/%EB%82%98%EC%9D%98%20%ED%9E%88%EC%96%B4%EB%A1%9C%20%EC%95%84%EC%B9%B4%EB%8D%B0%EB%AF%B8%EC%95%84%202%EA%B8%B0/1 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 21:00:24] "GET /images/to.png HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 21:00:25] "GET /detail/1/1?ani_path=%2Fuploads%2Fmy_hero_Academya2%2FOhys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 21:00:25] "GET /detail/1/jquery.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 21:00:25] "GET /detail/1/jquery.srt.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 21:00:25] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 21:00:25] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.vtt HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 21:00:25] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
122.47.22.201 - - [11/Apr/2018 21:00:25] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 21:00:26] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 21:00:47] "GET /upload HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 21:00:47] "GET /bootstrap/3.0.0/css/bootstrap.min.css HTTP/1.1" 404 -
Converting <Ohys-Raws_Boku_no_Hero_Academia_2_-_02_NTV_1280x720_x264_AAC.smi> OK!
Converting <Ohys-Raws_Boku_no_Hero_Academia_2_-_03_NTV_1280x720_x264_AAC.smi> OK!
122.47.22.201 - - [11/Apr/2018 21:05:51] "POST /uploaded HTTP/1.1" 200 -
119.192.54.107 - - [11/Apr/2018 21:05:51] "GET / HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 21:05:51] "GET /bootstrap/3.0.0/css/bootstrap.min.css HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 21:06:30] "GET / HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 21:06:30] "GET /images/to.png HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 21:06:31] "GET /detail/%EB%82%98%EC%9D%98%20%ED%9E%88%EC%96%B4%EB%A1%9C%20%EC%95%84%EC%B9%B4%EB%8D%B0%EB%AF%B8%EC%95%84%202%EA%B8%B0/1 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 21:06:31] "GET /images/to.png HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 21:06:32] "GET /detail/1/3?ani_path=%2Fuploads%2Fmy_hero_Academya2%2FOhys-Raws_Boku_no_Hero_Academia_2_-_02_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 21:06:32] "GET /detail/1/jquery.srt.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 21:06:32] "GET /detail/1/jquery.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 21:06:32] "GET /detail/1/jquery.srt.js HTTP/1.1" 404 -
122.47.22.201 - - [11/Apr/2018 21:06:33] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_02_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 200 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 21:06:33] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_03_NTV_1280x720_x264_AAC.vtt HTTP/1.1" 200 -
122.47.22.201 - - [11/Apr/2018 21:06:33] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_02_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
122.47.22.201 - - [11/Apr/2018 21:06:33] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_02_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
122.47.22.201 - - [11/Apr/2018 21:06:36] "GET /uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_02_NTV_1280x720_x264_AAC.mp4 HTTP/1.1" 206 -
Error on request:
Traceback (most recent call last):
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/ubuntu/flask_nas/venv/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe
 * Detected change in '/home/ubuntu/flask_nas/db.py', reloading
insert anime ok!
row is :
((66, u'\ub098\uc758 \ud788\uc5b4\ub85c \uc544\uce74\ub370\ubbf8\uc544 2\uae30', u'my_hero_Academya2', u'myheroaca.jpg', datetime.datetime(2018, 4, 11, 20, 22, 20)),)
66
row is :
((66, u'\ub098\uc758 \ud788\uc5b4\ub85c \uc544\uce74\ub370\ubbf8\uc544 2\uae30', u'my_hero_Academya2', u'myheroaca.jpg', datetime.datetime(2018, 4, 11, 20, 22, 20)),)
66
None
new row
1
insert detail sql success!
row is :
((66, u'\ub098\uc758 \ud788\uc5b4\ub85c \uc544\uce74\ub370\ubbf8\uc544 2\uae30', u'my_hero_Academya2', u'myheroaca.jpg', datetime.datetime(2018, 4, 11, 20, 22, 20)),)
66
1
run change_vtt.py from :/home/ubuntu/flask_nas/static/uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.vtt
66
1
is vtt: 
((40, 66, 1, u'/uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.mp4', None, 1),)
row is :
((66, u'\ub098\uc758 \ud788\uc5b4\ub85c \uc544\uce74\ub370\ubbf8\uc544 2\uae30', u'my_hero_Academya2', u'myheroaca.jpg', datetime.datetime(2018, 4, 11, 20, 22, 20)),)
66
1
not new row
2
insert detail sql success!
row is :
((66, u'\ub098\uc758 \ud788\uc5b4\ub85c \uc544\uce74\ub370\ubbf8\uc544 2\uae30', u'my_hero_Academya2', u'myheroaca.jpg', datetime.datetime(2018, 4, 11, 20, 22, 20)),)
66
2
run change_vtt.py from :/home/ubuntu/flask_nas/static/uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.vtt
66
2
is vtt: 
((41, 66, 2, u'/uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.mp4', None, 1),)
row is :
((1, u'\ub098\uc758 \ud788\uc5b4\ub85c \uc544\uce74\ub370\ubbf8\uc544 2\uae30', u'my_hero_Academya2', u'myheroaca.jpg', datetime.datetime(2018, 4, 11, 20, 49, 12)),)
1
2
not new row
3
insert detail sql success!
row is :
((1, u'\ub098\uc758 \ud788\uc5b4\ub85c \uc544\uce74\ub370\ubbf8\uc544 2\uae30', u'my_hero_Academya2', u'myheroaca.jpg', datetime.datetime(2018, 4, 11, 20, 49, 12)),)
1
3
run change_vtt.py from :/home/ubuntu/flask_nas/static/uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_02_NTV_1280x720_x264_AAC.vtt
1
3
is vtt: 
((42, 1, 3, u'/uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_02_NTV_1280x720_x264_AAC.mp4', None, 1),)
row is :
((1, u'\ub098\uc758 \ud788\uc5b4\ub85c \uc544\uce74\ub370\ubbf8\uc544 2\uae30', u'my_hero_Academya2', u'myheroaca.jpg', datetime.datetime(2018, 4, 11, 20, 49, 12)),)
1
3
not new row
4
insert detail sql success!
row is :
((1, u'\ub098\uc758 \ud788\uc5b4\ub85c \uc544\uce74\ub370\ubbf8\uc544 2\uae30', u'my_hero_Academya2', u'myheroaca.jpg', datetime.datetime(2018, 4, 11, 20, 49, 12)),)
1
4
run change_vtt.py from :/home/ubuntu/flask_nas/static/uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_03_NTV_1280x720_x264_AAC.vtt
1
4
is vtt: 
((43, 1, 4, u'/uploads/my_hero_Academya2/Ohys-Raws_Boku_no_Hero_Academia_2_-_03_NTV_1280x720_x264_AAC.mp4', None, 1),)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 100-660-140
66.102.6.174 - - [12/Apr/2018 13:53:56] "GET / HTTP/1.1" 200 -
66.102.6.174 - - [12/Apr/2018 13:53:56] "GET /upload HTTP/1.1" 200 -
66.102.6.19 - - [12/Apr/2018 13:53:56] "GET / HTTP/1.1" 200 -
66.102.6.174 - - [12/Apr/2018 13:53:56] "GET /favicon.ico HTTP/1.1" 404 -
66.102.6.174 - - [12/Apr/2018 13:53:56] "GET /favicon.ico HTTP/1.1" 404 -
66.102.6.23 - - [12/Apr/2018 13:53:56] "GET /favicon.ico HTTP/1.1" 404 -
 * Detected change in '/home/ubuntu/flask_nas/run.py', reloading
Exception in thread Thread-1 (most likely raised during interpreter shutdown):
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
  File "/usr/lib/python2.7/threading.py", line 763, in run
  File "/home/ubuntu/flask_nas/venv/local/lib/python2.7/site-packages/werkzeug/serving.py", line 702, in inner
  File "/home/ubuntu/flask_nas/venv/local/lib/python2.7/site-packages/werkzeug/serving.py", line 539, in serve_forever
  File "/usr/lib/python2.7/SocketServer.py", line 241, in serve_forever
  File "/usr/lib/python2.7/threading.py", line 585, in set
  File "/usr/lib/python2.7/threading.py", line 406, in notifyAll
<type 'exceptions.TypeError'>: 'NoneType' object is not callable
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 100-660-140
 * Detected change in '/home/ubuntu/flask_nas/run.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 100-660-140
220.181.159.77 - - [13/Apr/2018 02:34:03] code 400, message Bad request syntax ('JRMI\x00\x02K')
220.181.159.77 - - [13/Apr/2018 02:34:03] "JRMI K" 400 -
66.102.6.176 - - [13/Apr/2018 09:35:41] "GET / HTTP/1.1" 200 -
66.102.6.174 - - [13/Apr/2018 09:35:41] "GET /upload HTTP/1.1" 200 -
66.102.6.21 - - [13/Apr/2018 09:35:41] "GET / HTTP/1.1" 200 -
66.102.6.176 - - [13/Apr/2018 09:35:41] "GET /favicon.ico HTTP/1.1" 404 -
66.102.6.174 - - [13/Apr/2018 09:35:41] "GET /favicon.ico HTTP/1.1" 404 -
66.102.6.23 - - [13/Apr/2018 09:35:42] "GET /favicon.ico HTTP/1.1" 404 -
125.129.62.2 - - [13/Apr/2018 09:35:43] "GET / HTTP/1.1" 200 -
125.129.62.2 - - [13/Apr/2018 09:35:43] "GET /css/main.css?q=1522674725 HTTP/1.1" 304 -
125.129.62.2 - - [13/Apr/2018 09:35:44] "GET /images/to.png HTTP/1.1" 404 -
125.129.62.2 - - [13/Apr/2018 09:35:44] "GET /css/style.css?q=1522674725 HTTP/1.1" 304 -
125.129.62.2 - - [13/Apr/2018 09:35:44] "GET /images/myheroaca.jpg HTTP/1.1" 200 -
125.129.62.2 - - [13/Apr/2018 09:36:40] "GET /upload HTTP/1.1" 200 -
125.129.62.2 - - [13/Apr/2018 09:36:40] "GET /bootstrap/3.0.0/css/bootstrap.min.css HTTP/1.1" 404 -
Converting <Ohys-Raws_Boku_no_Hero_Academia_2_-_00_NTV_1280x720_x264_AAC.smi> OK!
Converting <Ohys-Raws_Boku_no_Hero_Academia_2_-_01_NTV_1280x720_x264_AAC.smi> OK!
125.129.62.2 - - [13/Apr/2018 09:40:04] "POST /uploaded HTTP/1.1" 200 -
125.129.62.2 - - [13/Apr/2018 09:40:04] "GET /bootstrap/3.0.0/css/bootstrap.min.css HTTP/1.1" 404 -
196.52.43.61 - - [13/Apr/2018 23:16:19] "GET / HTTP/1.0" 200 -
122.47.22.201 - - [14/Apr/2018 10:32:25] "GET /favicon.ico HTTP/1.1" 404 -
66.102.6.152 - - [14/Apr/2018 22:05:35] "GET /upload HTTP/1.1" 200 -
66.102.6.151 - - [14/Apr/2018 22:05:35] "GET / HTTP/1.1" 200 -
66.102.6.86 - - [14/Apr/2018 22:05:36] "GET / HTTP/1.1" 200 -
66.102.6.151 - - [14/Apr/2018 22:05:36] "GET /favicon.ico HTTP/1.1" 404 -
66.102.6.151 - - [14/Apr/2018 22:05:36] "GET /favicon.ico HTTP/1.1" 404 -
66.102.6.87 - - [14/Apr/2018 22:05:36] "GET /favicon.ico HTTP/1.1" 404 -
183.96.125.33 - - [15/Apr/2018 00:20:45] "GET /detail/2/jquery.js HTTP/1.1" 404 -
183.96.125.33 - - [15/Apr/2018 00:20:45] "GET /detail/2/jquery.srt.js HTTP/1.1" 404 -
183.96.125.33 - - [15/Apr/2018 00:20:46] "GET /favicon.ico HTTP/1.1" 404 -
